document-portal/document-portal-fuse.c

```diff
@@ -64,6 +64,18 @@
 #define ENODATA ENOATTR
 #endif
 
+#define XDP_XATTR_HOST_PATH "user.xdp-document-portal.host-path"
```

Comment:
Let's bikeshed on this a little :)

I think `user.` and `.host-path` are fine. I think the middle part should be either `document-portal`, or `portal.document` instead of `xdp-document-portal`.

For reference, the PipeWire streams we manage have `pipewire.access.portal.app_id` and `pipewire.access.portal.media_roles` on them.

Thoughts?

Reply:
I really don't care. Updated it to be `user.document-portal.host-path`.